cloudformation/external-slurmdbd/external_slurmdbd/external_slurmdbd_stack.py

```diff
@@ -361,3 +420,29 @@ def _add_cloudwatch_log_group(self):
             log_group_name=f"/aws/slurmdbd/{self.stack_name}",
             retention=logs.RetentionDays.ONE_WEEK,
         )
+
+    def _add_hosted_zone(self):
+        return route53.CfnHostedZone(
+            self,
+            id="ExternalSlurmdbdHostedZone",
```

Comment:
Shouldn't this be provided as parameter ?

Reply:
I will create an internal task to further dive deep on this.